src/CabanaPD_Particles.hpp

```diff
@@ -311,7 +320,9 @@ class Particles<DeviceType, PMB, Dimension>
     auto slice_vol() { return Cabana::slice<0>( _aosoa_vol, "volume" ); }
     auto slice_vol() const { return Cabana::slice<0>( _aosoa_vol, "volume" ); }
     auto slice_type() { return Cabana::slice<0>( _aosoa_other, "type" ); }
+    // Why some variable use two lines definition while others a single line?
     auto slice_W() { return Cabana::slice<1>( _aosoa_other, "strain_energy" ); }
+    // Velocities are vector: should they be defined as u, f, etc.?
```

Comment:
Remove before merge. Velocities were not (in my initial testing) performance critical and so were combined with the auxiliary particle properties